files/autopause/autopause-fcns.sh

```diff
@@ -17,26 +17,11 @@ rcon_client_exists() {
 }
 
 mc_server_listening() {
-  [[ -n $(netstat -tln | grep -e "0.0.0.0:$SERVER_PORT" -e ":::$SERVER_PORT" | grep LISTEN) ]]
+  mc-monitor status --host localhost --port $SERVER_PORT >& /dev/null
```

Comment:
I don't fully understand the suggestion since that's what mc-monitor status is doing. It's used already for health checks.

Reply:
...oh you mean skipping timeout?

Reply:
Yes, skipping the timeout, I couldn't get the timeout command line flag to work (`--retry-interval 1s --retry-limit 1 --timeout 1s` was not timing our with the stopped process). In the healthcheck the process is also checked, as the healthcheck was spawning endless processes initially.

Reply:
Now I see what you mean and yes, this is not going to be acceptable. I'll play with tweaking the knockd config, since running mc-monitor via the debugger externally causes the knock to kick in. This also makes me realize that the mc-monitor approach is really only useful during running state to detect zero players.

Reply:
imo, this won't be solved by `knockd`, this function is only used to switch from deamon init state to the initial timeout state, so the process isn't stopped here, it's waiting for the server to listen for connections (where exactly it does that during startup is another question, especially wrt modding)

Reply:
I didn't mean to imply knockd would solve this. I was just thinking out loud that I needed to temporarily change my container's config to let me recreate the blocked ping operation.

The server binds the listening port at the very end of startup, so it's a very robust (even with mods) way to know when the server is ready.

Reply:
FYI, have a PR open for the issue I found https://github.com/Raqbit/mc-pinger/pull/14

Reply:
my bad, didn't know that this failure to TO was strictly related to the paused process, I was only using mc-monitor from inside the image, Checking with the pure monitor docker image to a nonexisting server, I agree with this change and understand your previous comments.